src/main/java/com/codegym/jrugotom5/entity/User.java

```diff
@@ -1,4 +1,25 @@
 package com.codegym.jrugotom5.entity;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@Entity
+@Data
+@EqualsAndHashCode(exclude = "id")
 public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String firstName;
+
+    private String lastName;
```

Comment:
We name columns in db using underscore. Should add @Column(name = "fisrt_name") annotation to both variables.

Reply:
I can`t add relationship with the Advert entity because it doesn`t exist yet. I will do this when I add the Advert entity